flowermd/base/system.py

```diff
@@ -731,17 +731,17 @@ def __init__(
         edge=0.2,
         overlap=0.2,
     ):
+        self.density = density * u.g / (u.cm ** (3))
```

Comment:
Are there cases where the pack density is number density rather than mass density? Should we do the unit check (`g/cm**3` vs `m**(-3)`) here?

Reply:
Yeah, I think this is a good point. The whole idea is to never have to use mass density if your beads don't really have real masses, so we should make sure this works here too.